lib/charms/ovn_charm.py

```diff
@@ -1002,12 +1090,20 @@ def configure_bridges(self):
         else:
             opvs.remove('.', 'external_ids', 'ovn-bridge-mappings')
 
-        if self.options.prefer_chassis_as_gw:
-            opvs.set(
-                '.', 'external_ids:ovn-cms-options', 'enable-chassis-as-gw')
+        cms_opts = self._format_ovn_cms_options()
+        if cms_opts:
+            opvs.set('.', 'external_ids:ovn-cms-options', ','.join(cms_opts))
         else:
-            opvs.remove(
-                '.', 'external_ids', 'ovn-cms-options=enable-chassis-as-gw')
+            opvs.remove('.', 'external_ids', 'ovn-cms-options')
+
+    def _format_ovn_cms_options(self):
```

Comment:
To me the name of the method promises to format a ready to be used string, but instead it is offering a list of strings for further processing.  That is also fine, but I would expect a different name if that is what you want it to do.

Also, docstring please.